scrapper.py

```diff
@@ -35,7 +46,31 @@ def find_articles(self):
         """
         Finds articles
         """
-        pass
+        headers = {
```

Comment:
is there a better way to keep headers in a common place?

Reply:
I've decided to remove headers because I can access the site without them. But yes, if I needed them, I would keep them in constants.py